help/en/html/doc/Technical/JVMCapabilities.shtml

```diff
@@ -822,34 +811,32 @@
       </table>
 
       <a id="jdk17">
+      <a id="compatibilty">
       <h2>Running JMRI 5 on Java 17 or later</h2>
-      There are some known problems when running JMRI
-      on Java versions later than the recommended Java 11.
-      Specifically, for JMRI 5 on Java 17 these include:
+      There are some issues when using JMRI
+      on Java versions later than the required Java 11.
+      Specifically, for JMRI 5 these include:
 
       <ol>
         <li>There is no JavaScript (ECMAscript) interpreter available
+            in Java 15 or later
             so .js scripts can't be run within JMRI.  Note that this
             does not affect JavaScript run on a web browser, so that
             JMRI's webserver capability will still work OK.
-        <li>The module system prevents LibUSB (ch.ntb.usb) from accessing the hardware.
-            This is used in a few scripts. Not clear if its developers will
-            be issuing an update or not; possibly JMRI will have to
-            switch to a different library.
+            <p>
+            It’s possible that someday GraalVM or Rhino will provide a solution for this, 
+            but it’s still a ways in the future.  
+            The Windows and macOS versions of GraalVM are still tricky to install 
+            and for some older OSs aren’t available at all. 
+            Rhino doesn't yet provide full access to the JMRI Java classes.
```

Comment:
`’` should either be replaced with `'` or with `&rsquo;` at three places in this section.

Reply:
Done.  Thank you.